python/src/GaussianProcess_doc.i.in

```diff
@@ -82,6 +82,29 @@ isTrendStationary : bool
 
 // ---------------------------------------------------------------------
 
+%feature("docstring") OT::GaussianProcess::getSamplingMethod
+"Get the used method for getRealization.
+
+Returns
+-------
+samplingMethod : int
+    Used method for sampling.
+
+Notes
+-----
+
+Available parameters are :
+
+  * 0 (CHOLESKY) : Cholesky factor sampling (default method)
+
+  * 1 (HMAT) : H-Matrix method (if H-Mat available)
+
+  * 2 (GIBBS) : Gibbs method (in dimension 1 only)
```

Comment:
Github won't let me suggest the same change (adding the CHOLESKY/HMAT/GIBBS names) to `setSamplingMethod`, but it would be better to do it there also.

Reply:
The difference is that you could directly set `GaussianProcess.CHOLESKY`, `GaussianProcess.HMAT` or `GaussianProcess.GIBBS`

Reply:
This is why I think these names should be mentioned in the `setSamplingMethod` method doc, and not only in the `getSamplingMethod` doc.

The current `setSamplingMethod` doc is:

```
Set the used method for getRealization.

Available parameters are :
  * 0 : Cholesky factor sampling (default method)
  * 1 : H-Matrix method (if H-Mat available)
  * 2 : Gibbs method (in dimension 1 only)
```
The reader is not informed that `setSamplingMethod(ot.GaussianProcess.HMAT)` is a valid command.

Reply:
Your latest change is even better than what I had in mind. Thanks!